docs/probes.md

```diff
@@ -11,12 +11,20 @@ To do that, we rely on the qemu-guest-agent to be available inside the VM.
 
 A command supplied to an exec probe, will be wrapped by `virt-probe` in the operator and forwarded to the guest.
 
+## Guest-Agent Ping Probe
+
+Another option to probe the VM is by doing a qemu-guest-agent based `guest-ping`. This will ping the guest and return an error if the guest is not up and running.
+To easily define this on VM spec, specify `guestAgentPing: true` in VM's readiness probe spec; `virt-controller` will translate this into a corresponding command wrapped by `virt-probe`.
```

Comment:
I think it would be `guestAgentPing: {}` but I'd make it `guestAgentPing: { probe: true }` or something like this to make sure it does not cause problems with empty values etc.